From 8a1cf894a30820ff563ca12c727ba589ccc0d3b5 Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@freescale.com>
Date: Sun, 14 Apr 2013 19:54:53 +0530
Subject: [PATCH 061/124] fsl/usb: Add FSL USB Gadget entry in platform device
 id table

Add FSL USB Gadget entry in platform device id table

Signed-off-by: Suresh Gupta <suresh.gupta@freescale.com>
Change-Id: Iacef40798e9c0fe4e73da8ff871db9051fb30da3
Reviewed-on: http://git.am.freescale.net:8181/1302
Reviewed-by: Mehresh Ramneek-B31383 <ramneek.mehresh@freescale.com>
Reviewed-by: Li Yang-R58472 <LeoLi@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 drivers/usb/gadget/udc/fsl_udc_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/udc/fsl_udc_core.c b/drivers/usb/gadget/udc/fsl_udc_core.c
index 75ddf1a..5ab5657 100644
--- a/drivers/usb/gadget/udc/fsl_udc_core.c
+++ b/drivers/usb/gadget/udc/fsl_udc_core.c
@@ -2661,7 +2661,7 @@ static const struct platform_device_id fsl_udc_devtype[] = {
 	}, {
 		.name = "imx-udc-mx51",
 	}, {
-		/* sentinel */
+		.name = "fsl-usb2-udc"
 	}
 };
 MODULE_DEVICE_TABLE(platform, fsl_udc_devtype);
-- 
2.1.0.27.g96db324

